celer/lasso_fast.pyx

```diff
@@ -262,11 +278,11 @@ def celer(
                 # CAUTION: code does not yet include a best_theta.
                 # Can be an issue in screening: dgap and theta might disagree.
 
-                p_obj_in = primal(pb, alpha, Xw, y, w, weights)
+                p_obj_in = primal(pb, alpha, l1_ratio, Xw, y, w, weights)
                 gap_in = p_obj_in - highest_d_obj_in
 
                 if verbose_in:
-                    print("Epoch %d, primal %.10f, gap: %.2e" %
+                    print("Epoch %d, primal %.10f, gap: %.10e" %
```

Comment:
it's in scientific notation so 2 digits are enough (for debugging you can use more but don't merge as is)